backend/src/controllers/inventory.controller.js

```
@@ -0,0 +1,402 @@
+  res.status(200).json({
+    message: "customer deleted successfully",
+    success: true,
+    data: null,
+  });
+  try {
+  } catch (error) {
+    res
+      .status(500)
+      .json({ message: "Internal server error", success: false, data: null });
+    console.log(error.message);
+  }
```

Biome (2.1.2)

[error] This code is unreachable



(lint/correctness/noUnreachable)